### PR TITLE
Documentation header update

### PR DESCRIPTION
**Pull request by zhenik**
Cosmetic doc update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+    <p align="center">
+        <a href="https://github.com/fredrikhgrelland/vagrant-hashistack" alt="Built on">
+            <img src="https://img.shields.io/badge/Powered%20by%20-Vagrant--hashistack-orange?style=for-the-badge&logo=vagrant"/>
+        </a>
+    </p>
+</p>
+
+---
+
+Module contains a nomad job [./conf/nomad/presto.hc](./conf/nomad/presto.hcl) with [presto sql server](https://github.com/prestosql/presto).
```
Comment (Neha-Sinha2305) [inline README.md:18]:
```suggestion
Module contains a nomad job [./conf/nomad/presto.hcl](./conf/nomad/presto.hcl) with [presto sql server](https://github.com/prestosql/presto).
```

File: README.md [hunk 2]
```diff
@@ -194,7 +207,7 @@ presto --server localhost:8080 --catalog hive --schema default --user presto --f
 
 ## License
 
-________
+---
 
 ## References
```
Comment (Neha-Sinha2305) [inline README.md:212]: Table of contents does not include links to section that are present after **License** section.
Also, needs reordering.

**Review by Neha-Sinha2305 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -194,7 +207,7 @@ presto --server localhost:8080 --catalog hive --schema default --user presto --f
 
 ## License
```
Comment (Neha-Sinha2305) [inline README.md:208]: I do not find an explicit **License** section in [vagrant-hashistack-readme](https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/README.md). Do we need it?

**Review by Neha-Sinha2305 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (zhenik) [reply README.md:212]: I will move this section in example/resources/data dir

**Review by Neha-Sinha2305 — APPROVED** [review]

**Pull request merged**